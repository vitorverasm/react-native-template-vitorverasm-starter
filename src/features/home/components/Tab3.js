import React from 'react';
import { Text, View } from 'react-native';

const Tab3 = ({ props }) => (
  <View style={{ flex: 1, alignItems: 'center', justifyContent: 'center' }}>
    <Text>TAB 3</Text>
  </View>
);

export default Tab3;
