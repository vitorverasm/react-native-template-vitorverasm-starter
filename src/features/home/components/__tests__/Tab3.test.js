import { shallow } from 'enzyme';
import React from 'react';
import Tab3 from '../Tab3';

describe('Tab3.js', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<Tab3 />);
    expect(wrapper).toMatchSnapshot();
  });
});
