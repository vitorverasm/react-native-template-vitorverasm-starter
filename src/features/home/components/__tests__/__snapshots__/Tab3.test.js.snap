// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tab3.js renders correctly 1`] = `ShallowWrapper {}`;
